src/components/FallbackUIButton/index.test.tsx

```diff
@@ -15,7 +16,7 @@ describe('FallbackUIButton', () => {
   it('should render FallbackUIButton', async () => {
     render(<Wrapper />);
     const button: HTMLButtonElement = await screen.findByText('Get an error');
-    expect(button).toBeInTheDocument();
+    expect(button).toBeDefined();
```

Comment:
I don't understand why you change this, but this check is unnecessary, if you do not find an element, then there will be an error 

[testing-library.com/docs/queries/about](https://testing-library.com/docs/queries/about/)

Reply:
i'll fix this